moabb/paradigms/motor_imagery.py

```diff
@@ -125,8 +126,20 @@ def scoring(self):
         return 'accuracy'
 
 
+class MotorImagery(BaseMotorImagery):
+
+    def __init__(self, bp_low=8, bp_high=32, **kwargs):
```

Comment:
can you use fmin and fmax, to be consistent with MNE.

Reply:
fixed